src/mail/mjml/invitation.mjml

```diff
@@ -31,7 +31,7 @@
           <!-- Main Message -->
           <mj-text>
             {% blocktrans %}
-              {{username}} invited you as an {{role}} on the following document : 
+              {{username}} {{email}} invited you as an {{role}} on the following document : 
```

Comment:
{{email}} invited you with the role "{{role}}" on the following document :

Reply:
why do we want to add the email here?

Reply:
It's handy in order to know from which organization the person is coming from.
It's also a unique identifier. 
It's a good practice I catched from gdocs.